.travis.yml

```diff
@@ -1,66 +1,170 @@
     - secure: gfJ1/G69s0pOJhqoOB3bMAClvPPj3i++ckCCP/Yz2OMBFoAF3YCb2k2p5fRNX46s/hqkddHZnuSzaPRXHW5tToS7OhpN1HB6EDIfBS80zD1ZH5n5B8s/jKwzQLWv85bZATKc+R8Ir2ukBd/BQFurzd8JtQDpGS2hSCppOUQQApE3F+ghpET8dFPP7qncn1p98UBbbYyxiR53EM2xo93kByooe7nJiwjLvGVCp7zF9ZGKdp+yUzIb5tvOLnxvXrD2IvGl2Cuyf6bdqXUeM5yEhv0JwuD7SsIydCJZhTY8zCkMl3ITThY8HKV19sU6/zlG6aUsD5QAT58DiKk69icRc03kYCeLjUtuVHq/Nzq1grixc/uIi/KJlKxXd6h1NovAhLAkuuqjpLTKePOVeQdwCBZWQdXWPXUBK80px/ZYauydUqmaxHZq5JAM8hg0DbYICCqVcvro7WcUjCEW+VczO36wNU/qjbSsPehysIlWjt7qvcBfGPoGhlIm098jbXNQh0tM3PYInXiIPl/v1ctGUVWBlUkyppxgV3MYHFM04Pwv9lqovWmWuk+c4a22UE08/sTsHHrmS9T4zPfD6d/5Y1v8l8jMc9HGVa2ro642sRGgWS0QkazuQZ4ySYYpt98uv1NMq33L3ffbvsZo2mVCZ1ZWocCG7SbdWhJeOCysMMU=
-    - CLOUDFRONT_DISTRIBUTION_ID=E12YL8EZVABYR0
-    - BUCKET_NAME=dl.kuzzle.io
     - secure: MJlvmEbSYf5q7tQ/DtPTMGLpN5SgudJ03bpI9zqIjq5p55YosaRz4egXArTJeZsfNsA6Z9Wx2yV9/zqp1Iaq0O7AqoWuB08X5DJ0I3WsEHTgly2a94W+nC8QnESbMVuo7kkNlykq2Xo9w2oBmuSgMsPL+6Qlcbb9+h09hUgENISldnv9Hw7dSstfUIAXSGVp4D/RjwaEfoOEfORdSAxO5sr9PuxV8P8RSwKuu/M82dG582w/pY9MOq6goIaIWTbNQDTT2FS7hzCAZuvhLxPqMAkcNYt8v/BA0AVpnPcDCduC5T9LM1EkV61pizV2C6PSEf5x28HP4eK7flU8A82KBGwXUZjc93v0cuYAsYh3CDQcHm8spljgVakVmhs+rogOu6roHexEQR4Y6pkVH9s80Q7ZmANqKIShWheYIBScBS1dl4Nq8O9z7GRUtEwNjarDi7GzPJDumAsl53/qeDw+B5UN746QFqdIJDpvZw1lf92dAlKjbMtvwpxBoFjqBCACFDhIZFLZz+c+uC4SItgirLXSRjJMglzromo0MRQ0Ga4JNPcJVXamzwMT7U8d9b44CsesBhLx498xP8S3pr+LAHIeEuGnoaxFS6ACjDO92ffKPjYvQW2pubWILEYyIuUk/MfUIBIVNAzUWnLzRETKw8bD3B/mmikDlvwAvN2XdL8=
     - secure: epbUm0zRQdQmnZMazfI4B8wclxxN61Dfp45z9mhlNlZYfprTdMP/+cKvLrTzi4FbfRRkH9WwK3Ic1Wm5CQAvAHOVyVg9EYaMKVOhymd8bVsFWStChXmvBKw2ghV4/MsbhUAnP2mNA7YWDMnCsANV/g4JBNgSZaeWSw8hen2w5zugHRnUz9vzwLzSOa7vLsId8ctmjwwFAObvuc6U9Vy755FrmX86uDmZGE8oRQtiwbkmQpj15+AqhYLN/LUnyapxqFyT3bgKoc+TAzR1nrEDtZ3Yg4kkgvmlks7jXEvXhwSAvASQvGhvYNpBpDVsL9LP3oTvvzZIvAYpVXwUY9gjlbGw+oa2ysurDxZoguAK2Ni5ZhXF0KoGha1GfY4YeNguBkbainVuFbTXkiis/veGRPfYGYh1uStbn2WRDSeY7eeGfoVqLfWHuMUZIFZMXaTq41UEYLJwKFuNeNtxs6eJyfVybWaO52ZtZYb5heetk62joZZtZ1ph9hnsBdnAY9duKB2NE7QXXLpA2VPqqndKcm47/z3mHOadg4PlR/pOJno/TxPaxTRYVotPzt9k2DcGiAcu/kUpwsl4uXJ4xmYCnm18lsfJ2Sq/6mj0+bguZLVTyt3j3uL3IVV+9sPFYwNod4fPEeH5xGvk0Vqk/k+IX4db5Tz/UK+7EV+Ii19XUEY=
+    - AWS_CLOUDFRONT_DISTRIBUTION_ID_DL=E12YL8EZVABYR0
+    - AWS_CLOUDFRONT_DISTRIBUTION_ID_PROD=E2GPBJ9T0QR37G
+    - AWS_CLOUDFRONT_DISTRIBUTION_ID_DEV=E35G0B414M3IWU
+    - AWS_S3_BUCKET_DL=dl.kuzzle.io
+    - AWS_S3_BUCKET_PROD=console.kuzzle.io
+    - AWS_S3_BUCKET_DEV=next.console.kuzzle.io
+
+stages:
+  - tests
+  - build & deploy
+
+jobs:
+  include:
+# ---------------------------------------
+# Unit Testing
+# ---------------------------------------
+    - stage: tests
+      name: "Unit Tests"
+      script:
+        - npm install
+        - npm run unit
+
+      after_script:
+        - npm run codecov
+
+# ---------------------------------------
+# Functionnal Testing
+# ---------------------------------------
+    - stage: tests
+      name: "Functionnal Tests"
```

Comment:
```suggestion
      name: "End to end Tests"
```

Reply:
Yep, I'll fix that :slightly_smiling_face: